functions/supplier/twitter-supplier/src/main/java/org/springframework/cloud/fn/supplier/twitter/status/search/TwitterSearchSupplierConfiguration.java

```diff
@@ -56,9 +56,6 @@
 	@Autowired
 	private Twitter twitter;
 
-	@Autowired
-	private SearchPagination searchPage;
```

Comment:
This was causing circular reference as TwitterSearchSupplierConfiguration needs it's SearchPagination field injected, but wait, who will provide that to it? Ahh, the TwitterSearchSupplierConfiguration will supply it in https://github.com/spring-cloud/stream-applications/pull/213/files#diff-68c351c4c53dd25a5a570e9483de5205777402f7059498816f6a520bd58038dbR63. Ok, so let's create one of those TwitterSearchSupplierConfiguration , etc....

IOW, the configuration that creates the bean can not also inject it onto itself.